src/Defaults.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace ipl\Orm;
+
+use IteratorAggregate;
+use Traversable;
+
+class Defaults implements IteratorAggregate
+{
+    /** @var array<string, mixed> Registered defaults */
+    protected $defaults = [];
+
+    /**
+     * Iterate over the defaults
+     *
+     * @return Traversable
+     */
+    public function getIterator(): Traversable
+    {
+        foreach ($this->defaults as $column => $default) {
+            yield $column => $default;
+        }
+    }
+
+    /**
+     * Add a default for the given property
+     *
+     * @param string $property
+     * @param mixed $default If it's a closure, its interface is assumed to be
+     *                       ({@see Model} $subject, string $propertyName)
```

Comment:
Looking at the phpDocumentor documentation I'd rather use [@see](https://docs.phpdoc.org/guide/references/phpdoc/tags/see.html) instead of [@link](https://docs.phpdoc.org/guide/references/phpdoc/tags/link.html), as the latter doesn't support structural elements explicitly.

Reply:
Good to know, I've always used `@link` and now I have a good reason to use `@see` too! 😆